### ISSUE TITLE
The target "GitVersion" does not exist in the project.

### ISSUE THREAD

**Issue by jhalbrecht** [xamarin/mqtt#125]
I'm attempting to evaluate "Hermes" mqtt for Xamarin.Forms.

I get error `The target "GitVersion" does not exist in the project.` on line 26 of "Directory.Build.targets" `<Target Name="SetVersions" BeforeTargets="GetAssemblyVersion" DependsOnTargets="GitVersion" Returns="$(Version)">`  

I cloned and then attempted to build using 
```
Microsoft Visual Studio Professional 2017  (VS2017rc)
Version 15.4.1
VisualStudio.15.Release/15.4.1+27004.2005
Microsoft .NET Framework
Version 4.7.02556

Installed Version: Professional
[...]
Xamarin   4.7.10.22 (fe36bec)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin.Android SDK   8.0.0.33 (HEAD/7a6a056e8)
Xamarin.Android Reference Assemblies and MSBuild support.

Xamarin.iOS and Xamarin.Mac SDK   11.2.0.8 (9a9f054)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.
```

**Comment by kzu** [comment]
Did you do an `msbuild /t:restore` first?

**Comment by jhalbrecht** [comment]
Are there build instructions for this project? After reading the README.md and ReleaseNotes.md I saw the .sln and loaded up VS2017... 

Now I tried;
I opened a VS command line prompt and issued `msbuild /t:restore` which seemed to run without error. I went back to vs and built the project. Same errors. Then back to the command line and ran `msbuild` itself which appeared to build the client and server but failed on the 'tests' and 'integration tests' when I built from vs.

Looks like I have enough to test the client. I had that all the time with `Install-Package System.Net.Mqtt -Pre`

I was hoping for some client examples.
